snapd-glib/snapd-client-sync.c

```diff
@@ -519,6 +519,36 @@ snapd_client_get_interfaces_sync (SnapdClient *client,
     return snapd_client_get_interfaces_finish (client, data.result, plugs, slots, error);
 }
 
+/**
+ * snapd_client_get_interface_info_sync:
+ * @client: a #SnapdClient.
+ * @names: (array zero-terminated=1): a null-terminated array of interface names or %NULL.
+ * @flags: control what information is returned about the interfaces.
+ * @cancellable: (allow-none): a #GCancellable or %NULL.
+ * @error: (allow-none): #GError location to store the error occurring, or %NULL to ignore.
+ *
+ * Get the installed snap interfaces.
```

Comment:
Could you add a comment comparing this to get_interfaces () and one in get_interfaces () that does the same? In particular I'd like to steer users to choosing the correct API given they have such similar names.